docs/user/nifi/avro.rst

```diff
@@ -1,17 +1,14 @@
 Avro Processors
 ---------------
 
-GeoAvro processors (``AvroToPutGeoMesa``) accept the following configuration parameters for specifying the
+GeoAvro processors (``AvroToPutGeoMesa*``) accept the following configuration parameters for specifying the
 input source. Each datastore-specific processor also has additional parameters for connecting to the datastore,
 detailed in the following sections.
 
-+-------------------------+-------------------------------------------------------------------------------------------+
-| Property                | Description                                                                               |
-+=========================+===========================================================================================+
-| ``Avro SFT match mode`` | Determines how Avro SimpleFeatureType mismatches are handled.                             |
-+-------------------------+-------------------------------------------------------------------------------------------+
-
-The SimpleFeatureTypes in GeoAvro may or may not match the SimpleFeatureType in the target datastore.
-To address this, the AvroToPut processors have a property to set the SFT match mode. It can either be set to
-an exact match ("by attribute number and order") or a more lenient one ("by attribute name"). The latter setting
-will not write fields which are not in the target SFT.
```

Comment:
Do the AvroToPutGeoMesa* Processors inherit the schema compatibility modes from the converter processors?

If so, should the docs link to them?  (Thanks for answering all these questions!  I find this to be some subtle stuff we are keeping track of.)

Reply:
they use the same property, but they are handled slightly differently... i'd have to go check exactly how the converters handle `existing` mode. i'm not sure a cross-link would be useful though, other than in a "look you can also use converters" way.